src/components/components/ebay-notice-base/index.marko

```diff
@@ -51,6 +54,12 @@ $ var prefixClass = input.prefixClass;
                     class=input.iconClass
                 />
             </else-if>
+            <else-if(status === 'education' && input.type === 'section')>
```

Comment:
If this is only available for section notices, should it be in `notice-base`? Maybe this should be moved to `ebay-section-notice` instead, or this is a hint that we need to do some refactoring somewhere

Reply:
Definitely need a refactor here. We just want to release this for now and may be clean up in next major release.

Reply:
I was also initially concerned that we might end up introducing too many conditionals. If it's starting to feel unwieldy. please consider a separate ebay-education-notice. If so, we might also need to split it out on the skin side too (for parity).

Reply:
This conditional and the other one for `input.type === 'education'` have me convinced that Ian is right and we should create a separate `ebay-education-notice`. I know it's not ideal and we'd be scrapping a lot of work that we already did, but if we don't make the switch now it will be a breaking change in the future.

Reply:
As discussed, If we need to refactor this to separate component without notice-base, we are totally violating DRY here. If we want to create a dedicated story to improve DX, its pretty much same code with new story file and readme(as education is still a section notice under the hood). I am not sure doing either of the above yield anything other than extra work for us. Hence will leave this as is for now and we shall revisit when we are planning to make icon as a dedicated component.